lib/github-pages-health-check/error.rb

```diff
@@ -33,6 +33,10 @@ def message_with_url
       end
       alias_method :message_formatted, :message_with_url
 
+      def to_s
+        "#{message_with_url} (#{self.class.name.split('::').last})".gsub("\n", " ").squeeze(" ").strip
```

Comment:
Ahh. Meant to add one: `The custom domain for your GitHub Pages site is pointed at an outdated IP address. You must update your site's DNS records if you'd like it to be available via your custom domain. For more information, see https://help.github.com/articles/setting-up-a-custom-domain-with-github-pages/.  (DeprecatedIPError)`